src/main/scala/org/broadinstitute/dsde/firecloud/model/Library.scala

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.firecloud.model
+
+import spray.json.JsObject
+
+/**
+  * Created by ahaessly on 11/2/16.
+  */
+
+  case class LibrarySearchParams(
+    searchTerm: String,
+    from: Int = 0,
+    size: Int = 10)
+
+  case class LibrarySearchResponse(
+    total: Int,
+    results: Array[JsObject])
```

Comment:
let's brainstorm this. My gut tells me we should use something more strongly typed than JsObject here, for consumption by Scala code. This class might need a constructor (read: `apply` method) that accepts a JsObject, and/or accepts a SearchHit object.

The parallel to the `Document` model class is interesting. `Document` is intended to be created by Scala and sent to ES, so it makes sense for it to hold a JsObject, because ES expects a JsObject. `LibrarySearchResponse` is intended to be read *from* ES and consumed by Scala, so it makes sense for it to hold a more strongly-typed object.